### PR TITLE
CNN Evaluation (aka DeepCluster)

### PR DESCRIPTION
**Pull request by nhaubrich**
This contains the minimal working CNN evaluation in the analyzer. 
 **A summary**:
 - Save the keras model as a tensorflow graph (DeepCluster.pb, and I've included the script here for reference)
 - Include CMSSW tensorflow in analyzer
 - Get pixel image from cluster by centering image on chagre centroid and normalizing
 - Feed image (and other variables) into tensorflow graph

**Still to do:**
 - Verify evaluation is the same (x-y in analyzer could be screwed up relative to training)
 - Clean up debug statements
 - Put evaluation in its own organized bit of code and evaluate pion and shared hits too
 - Use a real training (current one was trained on poor statistics)
 - See if evaluation works in crab jobs (relative path for loading DeepCluster.pb might break)

**Other points of discussion:**
 - Is there a better name than DeepCluster?
 - Should all the training (i.e. MergedHits repo) be moved to this ML folder for integration?

### CODE REVIEW

**Comment by nhaubrich** [comment]
- Evaluation is now consistent between training script and analyzer
 - Switched postprocessor to normalized charge for now

**Comment by nhaubrich** [comment]
- Updated CNN with actual training (signal=shared hits, normalized images + totADC/10000, 30 epochs)
 - Renamed ML to 'ml'
